pageObjects/NotesPage.js

```diff
@@ -19,7 +19,7 @@ class NotesPage extends BasePage {
         this.newNoteTitleField.sendKeys(title)
         element(by.buttonText('Save')).click()
         if (title !== '' && body !== '') {
-            browser.wait(EC.presenceOf(this.noteFirst), browser.params.customTimeout,
+            browser.wait(EC.elementToBeClickable(this.noteFirst), browser.params.customTimeout,
```

Comment:
Looks like _EC.elementToBeClickable_ also doesn't guarantee that _this.noteFirst_ was created. What do you think about it @Xotabu4 ?